scripts/_common.sh

```diff
@@ -5,10 +5,7 @@
 #=================================================
 
 # Debian package version for Grafana
-GRAFANA_VERSION="10.1.2"
-
-# dependencies used by the app
-pkg_dependencies="influxdb"
+GRAFANA_VERSION="10.2.1"
```

Comment:
```suggestion
GRAFANA_VERSION="$(ynh_read_manifest --key='version' | | cut -d~ -f1)"
```

Reply:
I can't see where this variable `$grafana_version` is being used in the code

also the CI displays a warning...
```
45722 WARNING _common.sh: command substitution: line 9: syntax error near unexpected token `|'
45723 WARNING _common.sh: command substitution: line 9: `ynh_read_manifest --key='version' | | cut -d~ -f1)"'
```

Reply:
Hmm if I recall correctly, the package version is  incremented to enable upgrading from the Debian repo (otherwise the package would never be updated), and that shell variable was somehow required by the automatic PR creator... That's maybe not necessary any more.

Reply:
ah, I found it in master, in the `updater.sh` script. Was this script working/updating the app? shall we put it back?
https://github.com/YunoHost-Apps/grafana_ynh/blob/f7b8924af0ba32f616ca4b1cce35499abe0a5395/.github/workflows/updater.sh#L51

Reply:
I guess the original idea was to install/upgrade with a specific version from the temporary debian repo. But I think it never worked (i‧e. always installed the latest version).